R/get_tables.R

```diff
@@ -1,21 +1,22 @@
-#' Gets the available tables
+#' List the available tables on the connection
 #'
 #' @template conn
-#' @param pattern A regex pattern with which to subset the returned tables
-#' @param show_temporary (`logical`)\cr
+#' @param pattern (`character(1)`)\cr
+#'   Regex pattern with which to subset the returned tables.
+#' @param show_temporary (`logical(1)`)\cr
 #'   Should temporary tables be listed?
+#' @return
+#'   A data.frame containing table names in the DB.
```

Comment:
Including schema and catalog when relevant? Or information that uniquely defines all available tables?